indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/models/IndexedStorageManifest.scala

```diff
@@ -3,18 +3,66 @@ package uk.ac.wellcome.platform.archive.indexer.bags.models
 import java.time.Instant
 
 import io.circe.generic.extras.JsonKey
-import uk.ac.wellcome.platform.archive.common.storage.models.StorageManifest
+import uk.ac.wellcome.platform.archive.common.storage.models.{
+  StorageManifest,
+  StorageManifestFile
+}
+
+case class IndexedSuffixTally(
+  suffix: String,
+  count: Int
+)
+
+case class IndexedFileFields(
+  path: String,
+  name: String,
+  size: Long,
+  checksum: String,
+  @JsonKey("type") ontologyType: String = "File"
+)
+
+object IndexedFileFields {
+  def apply(storageManifestFile: StorageManifestFile): IndexedFileFields = {
+    IndexedFileFields(
+      path = storageManifestFile.path,
+      name = storageManifestFile.name,
+      size = storageManifestFile.size,
+      checksum = storageManifestFile.checksum.value
+    )
+  }
+}
 
 case class IndexedStorageManifest(
   id: String,
+  space: String,
+  version: Int,
   createdDate: Instant,
+  payloadFiles: Seq[IndexedFileFields],
+  payloadFileCount: Int,
+  payloadFileSize: Long,
+  payloadFileSuffixTally: Seq[IndexedSuffixTally],
```

Comment:
Minor: Looking at this written out, I wonder if this could be devolved into another case class? Say:

```scala
case class PayloadStats(
  fileCount: Int,
  fileSize: Long,
  fileSuffixTally: Seq[IndexedSuffixTally]
)
```

and then that appears under `payloadStats` and `newPayloadStats` on the IndexedStorageManifest object? To reduce the repetition a bit?

Reply:
Will follow in future PR.